news/_posts/2020-07-08-how-bug-reports-are-handled.md

```diff
@@ -0,0 +1,59 @@
+
+The maintainers team really wants to improve the issue processing workflow, and for that will soon become stricter. In fact, a lot of big Open Source projects are already pretty strict about issues: if you don’t respect their template, your issue is most likely to be closed!
+
+This is a good thing: by filtering out all the low quality issues, we ensure that developers can focus on the interesting ones, the issues that can bring the most value to the project and improve the quality of PrestaShop as a whole. It also speeds up the whole process by making sure that they can immediately start working on an issue while wasting as little time as possible reproducing, investigating, verifying the environment/the modules, etc.
+
+**We will progressively start to enforce these rules by refusing and even closing bug reports that do not provide the information required in the template**, and in particular the mandatory information we need to verify it: reproduction steps, PrestaShop version, PHP version, etc.
+
+**This obviously is not an absolute**, and if your issue doesn’t need reproduction steps the team will not blindly close it! However, this will help us make sure we can immediately start to work on the issue.
+
+Please remember that if your issue is closed because of these new rules, it is NOT because the team doesn’t care or doesn’t want PrestaShop to improve: it’s about optimizing resources. We will be happy to review your issue if you open a new one with all the information we need to reproduce.
+
+# Some examples
+
+## :scream: Bad bug reports
+
+- **Clearly incomplete**: you forgot to include some mandatory information like the PrestaShop version, the exact problem you encountered, or even the steps to reproduce!
+- **Wrong use of Github**: the team created a few templates to help you provide the information that’s typically needed to reproduce an issue. If you delete the whole template, we most likely won’t have this information and will lose time by asking and waiting for an answer. Same thing if you don’t write your issue in English, or just paste a 200 lines long log message with zero details.
+- **Request for support**: Bug reports should only be used to report issues which are caused by an error in the software. If the problem you report lies in your own configuration, or environment, or a module/theme that doesn’t belong to the project, the team will not be able to help you. You can use our [dedicated support](https://www.prestashop.com/fr/support) for this kind of issue!
```

Comment:
Maybe I should add a link to the forum and the Slack too ?

Reply:
to the slack would be good, yes.
https://github.com/PrestaShop/open-source/tree/master/slack

Reply:
Tell me if that's OK for you now :)

Reply:
👍